spec/controllers/houses_controller_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe HousesController, type: :controller do
+  include Devise::TestHelpers
+  
+  describe "GET index" do
+    let(:user) { User.create!(email: "user@bloccit.com", password: "helloworld") }
+    let (:houses) {{address: 'Levittown', cost: '400000', period: '30', down_payment: '100000', 
```

Comment:
This is not doing anything.

`House` is a model.  You will need to create `house`s for the `user`.  Meaning database records of the `House` table, for that `user`.